impl/maven-core/src/main/java/org/apache/maven/internal/impl/SisuDiBridgeModule.java

```diff
@@ -253,11 +244,6 @@ private <Q> Q getInstance(Binding<Q> binding) {
             return compile(binding).get();
         }
 
-        private static Comparator<Binding<?>> getBindingComparator() {
```

Comment:
As suggested above, a good alternative could be to move this method to `Binding` to make it reusable.

Reply:
good idea thx.